source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/APIControllers/Groups/GetGroupsTest.cs

```diff
@@ -0,0 +1,159 @@
+                WebsiteUrl = "https://testorg2.org",
+                BuildEngineUrl = "https://buildengine.testorg2",
+                BuildEngineApiAccessToken = "replace",
+                OwnerId = CurrentUser.Id
+
+            });
+            org3 = AddEntity<AppDbContext, Organization>(new Organization
+            {
+                Name = "TestOrg3",
+                WebsiteUrl = "https://testorg3.org",
+                BuildEngineUrl = "https://buildengine.testorg3",
+                BuildEngineApiAccessToken = "replace",
+                OwnerId = CurrentUser.Id
+
+            });
+            CurrentUserMembership = AddEntity<AppDbContext, OrganizationMembership>(new OrganizationMembership
+            {
+                UserId = CurrentUser.Id,
+                OrganizationId = org1.Id
+            });
+            CurrentUserMembership2 = AddEntity<AppDbContext, OrganizationMembership>(new OrganizationMembership
+            {
+                UserId = CurrentUser.Id,
+                OrganizationId = org2.Id
+            });
+            group1 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup1",
+                Abbreviation = "TG1",
+                OwnerId = org1.Id
+            });
+            group2 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup2",
+                Abbreviation = "TG2",
+                OwnerId = org1.Id
+            });
+            group3 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup3",
+                Abbreviation = "TG3",
+                OwnerId = org2.Id
+            });
+            group4 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup4",
+                Abbreviation = "TG4",
+                OwnerId = org3.Id
+            });
+            groupMembership1 = AddEntity<AppDbContext, GroupMembership>(new GroupMembership
+            {
+                UserId = CurrentUser.Id,
+                GroupId = group1.Id
+            });
+        }
+
+        [Fact]
+        public async Task Get_Groups_For_An_OrganizationHeader()
+        {
+            BuildTestData();
+
+            var url = "/api/groups" ;
+            var response = await Get(url, org1.Id.ToString());
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            var groups = await DeserializeList<Group>(response);
+
+            Assert.Equal(2, groups.Count);
+
+            var ids = groups.Select(g => g.Id);
+
+            Assert.Contains(group1.Id, ids);
+            Assert.Contains(group2.Id, ids);
+            Assert.DoesNotContain(group3.Id, ids);
+            Assert.DoesNotContain(group4.Id, ids);
+        }
+        [Fact] async Task Get_With_No_Organization()
+        {
+            BuildTestData();
+            var url = "/api/groups";
+            var response = await Get(url); // Empty string for org header
```

Comment:
empty string? I see no empty string

Reply:
The test Get method defaults the organization to an empty string if you don't specify an organization

Reply:
right, but the comment is describing behavior that isn't written here. (there is no empty string present)

Reply:
Comment updated